Traducir.Api/Controllers/StringsController.cs

```diff
@@ -217,5 +217,40 @@ void composePredicate(Func<SOString, bool> newPredicate)
             return BadRequest();
         }
 
+        [HttpDelete]
+        [Route("app/api/delete-suggestion")]
+        public async Task<IActionResult> DeleteSuggestion([FromQuery] DeleteSuggestionViewModel model)
+        {
+            //Have to validate everything before continuing
+            var str = await _soStringService.GetStringByIdAsync(model.StringId);
```

Comment:
Instead of getting the string from the cache, you could implement a method on the service that queries the database. The cache is an easy way to get a string, but I don't think it makes sense to either extend it to support suggestions or to pass the stringId here to fetch the string.